tendrl/commons/jobs/__init__.py

```diff
@@ -210,8 +210,8 @@ def process_job(job):
             logger.log(
                 "info",
                 NS.publisher_id,
-                {"message": "Running Flow %s" %
-                            job.payload['run']},
+                {"message": "Running %s" %
+                            job.payload['run'].rpartition('.')[2]},
```

Comment:
please use simple python code, what is the guarantee of rpartition and index 2 being always present? 

job.payload['run'].split(".")[-1]